pkg/cmd/version.go

```diff
@@ -38,6 +42,38 @@ func NewVersionCmd() *cobra.Command {
 		go compiler : %s
 		platform    : %s/%s
 `, version, buildDate, runtime.Version(), runtime.Compiler, runtime.GOOS, runtime.GOARCH)
+			if NewVersionAvailable() {
+				fmt.Println("New version of Gardenctl is available at https://github.com/gardener/gardenctl/releases/latest")
+				fmt.Println("Please get latest version from above URL and see https://github.com/gardener/gardenctl#installation for how to upgrade")
+			}
 		},
 	}
 }
+
+//NewVersionAvailable returns whether new version is available
+func NewVersionAvailable() bool {
+	var latestVersion string
+	gardenctlLatestURL := "https://api.github.com/repos/gardener/gardenctl/releases/latest"
+	resp, err := http.Get(gardenctlLatestURL)
+	if err != nil {
+		print(err)
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		print(err)
+	}
+
+	data := make(map[string]interface{})
+	err1 := json.Unmarshal([]byte(body), &data)
+	if err1 != nil {
+		print(err1)
+	}
+	if data["tag_name"] != nil {
+		latestVersion = data["tag_name"].(string)
+	}
+	if version_tool.CompareSimple(latestVersion, version) == 1 {
```

Comment:
Why we use another package for version comparison when there is `github.com/Masterminds/semver` and it is used in g/g?